README.md

```diff
@@ -85,7 +96,7 @@ Call the launchApp() method from the test case with relevant configuration to se
 
 ```swift
 import XCTest
-import AutomationTools
+import AutomationTools_Core
```

Comment:
Should this not be - instead of _?

Reply:
Same for the next snippet below this?

Reply:
Good one! Actually it's intentional, module names are generated with the underscore.

Reply:
👍